app/graphql/resolvers/billable_metrics_resolver.rb

```diff
@@ -10,13 +10,21 @@ class BillableMetricsResolver < GraphQL::Schema::Resolver
     argument :ids, [String], required: false, description: 'List of plan ID to fetch'
     argument :limit, Integer, required: false
     argument :page, Integer, required: false
+    argument :recurring, Boolean, required: false
     argument :search_term, String, required: false
 
     argument :aggregation_types, [Types::BillableMetrics::AggregationTypeEnum], required: false
 
     type Types::BillableMetrics::Object.collection_type, null: false
 
-    def resolve(ids: nil, page: nil, limit: nil, search_term: nil, aggregation_types: nil)
+    def resolve( # rubocop:disable Metrics/ParameterLists
```

Comment:
Maybe instead of disabling the linter, you could turn it into `def resolve(**args)`

And change the call to the query with:

```ruby
result = ::BillableMetricsQuery.new(organization: current_organization).call(
  search_term: args[:search_term],
  page: args[:page],
  limit: args[:limit],
  filters: args.slice(%i[ids recurring aggregation_types]),
)
```